docs/developer-guide/test-e2e.md

```diff
@@ -1,21 +1,21 @@
 # E2E Tests
 
-The directory contains E2E tests and test applications. The test assume that Argo CD services are installed into `argocd-e2e` namespace or cluster in current context. One throw-away
-namespace `argocd-e2e***` is created prior to tests execute. The throw-away namespace is used as a target namespace for test applications.
+The test [directory](../../test) contains E2E tests and test applications. The tests assume that Argo CD services are installed into `argocd-e2e` namespace or cluster in current context. A throw-away
+namespace `argocd-e2e***` is created prior to the execution of the tests. The throw-away namespace is used as a target namespace for test applications.
 
-The `test/e2e/testdata` directory contains various Argo CD applications. Before test execution directory is copies into `/tmp/argocd-e2e***` temp directory and used in tests as a
+The [/test/e2e/testdata](../../test/e2e/testdata) directory contains various Argo CD applications. Before test execution directory is copies into `/tmp/argocd-e2e***` temp directory and used in tests as a
```

Comment:
The links to github directories will fail in readthedocs: https://argo-cd--20335.org.readthedocs.build/en/20335/developer-guide/test-e2e/

If you want to link to them, use the full GitHub URL.

Reply:
good point, let me update that